product_docs/docs/pgd/5/node_management/node_types.mdx

```diff
@@ -4,19 +4,16 @@ title: Node types
 
 ## Data nodes
 
-A data node in PGD is a node which runs a Postgres instance. It replicates data to all other data nodes. It also participates in the cluster wide Raft decision making around locking and leadership. It will be the member of one or more groups and by default will be a member of the "top level" group which spans all data nodes in the cluster.
+A data node in PGD is a node that runs a Postgres instance. It replicates data to all other data nodes. It also participates in the cluster-wide Raft decision-making around locking and leadership. It will be the member of one or more groups and by default will be a member of the "top level" group that spans all data nodes in the cluster.
 
 ## Witness nodes
 
-A witness node behaves like a data node in that it participates in the cluster wide Raft decision making around locking and leadership. It does not replicate or store data though. The purpose of a witness node is to be available to ensure a majority can be achieved when the cluster seeks a consensus. The [Witness nodes](witness_nodes) section has more details.
+A witness node behaves like a data node in that it participates in the cluster-wide Raft decision-making around locking and leadership. It doesn't replicate or store data, though. The purpose of a witness node is to be available to ensure a majority can be achieved when the cluster seeks a consensus. [Witness nodes](witness_nodes) has more details.
 
 ## Logical standby nodes
 
-Logical standby nodes are nodes which recieve the logical data changes from another node and replicate them locally. They can be used to replace the node they are replicating, if that node should become unavailable, with some caveats. See [Logical standby nodes](logical_standby_nodes) for more details.
+Logical standby nodes are nodes that receive the logical data changes from another node and replicate them locally. You can use a logical standby node to replace the node it's replicating if that node becomes unavailable, with some caveats. See [Logical standby nodes](logical_standby_nodes) for more details.
```

Comment:
Since the user would not do anything manually here, but the system would handle all operations, I would say the subject here is PGD. 
```suggestion
Logical standby nodes are nodes that receive the logical data changes from another node and replicate them locally. PGD can use a logical standby node to replace the node it's replicating if that node becomes unavailable, with some caveats. See [Logical standby nodes](logical_standby_nodes) for more details.
```